src/components/TradeWidget/TokenRow.tsx

```diff
@@ -286,7 +286,7 @@ const TokenRow: React.FC<Props> = ({
               )}
             </>
           )}
-          {!readOnly && isWeth && <WrapEtherBtn label="+ Wrap Ether" />}
+          {editableAndConnected && isWeth && <WrapEtherBtn label="+ Wrap Ether" />}
```

Comment:
`Deposit` and `Wrap Ether` buttons are unaligned because one of them isn't a button
L266: `+ Deposit` is a `<div>` for some reason

Reply:
its fixed now